packages/legend-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -88,6 +88,11 @@ const EXCLUSIONS: { [key: string]: ROUNTRIP_TEST_PHASES[] | typeof SKIP } = {
   'nested-embedded-relational-mapping.pure': SKIP,
   'relational-mapping-filter.pure': SKIP,
 
+  // Needs a fix on engine. Engine shouldn't produce `source` for pure property mapping and
+  // relational property mapping beacuse they can be resolved at compilation.
+  'pure-property-mapping-local-property.pure': SKIP,
```

Comment:
We should only skip the `ROUNTRIP_TEST_PHASES.PROTOCOL_ROUNDTRIP` phase. The test should still compile and compare the grammar.